docker-compose.yml

```diff
@@ -1,7 +1,7 @@
 version: "3"
 services:
   elasticsearch:
-    image: docker.elastic.co/elasticsearch/elasticsearch:7.7.0
+    image: docker.elastic.co/elasticsearch/elasticsearch:7.6.1
```

Comment:
Er det en grunn for å endre versjon tilbake?

Reply:
eeh, nei. Jeg bare slettet den gamle docker-compose fila og kopierte `docker-compose.contract.yml` hit. Kan oppdatere den til å bruke 7.7.0

Reply:
f81be82